### ISSUE TITLE
how do i set the "Device Type String" that shows on the device settings tab?

### ISSUE THREAD

**Issue by mnsandler** [HomeSeer/Plugin-SDK#119]
i cant find this in the documentation

i just need a code snippet

**Comment by wvuyk** [comment]
I am using this definition for the device type items, including the description (in VB):

```
 Dim DT As New Devices.Identification.TypeInfo With {
                .ApiType = Devices.Identification.EApiType.Device,
                .Type = Devices.Identification.EDeviceType.Generic,
                .SubType = PluginDeviceSubType.SyncBox,
                .SubTypeDescription = PluginDeviceSubType.SyncBox.Description
                }
```

**Comment by mnsandler** [comment]
thanks for the sample. which property allows me to assign my text string like "Dual-band Switch", "Keypadlinc", etc

**Comment by spudwebb** [comment]
The property allowing you to assign the Device Type String is supposed to be Devices.Identification.TypeInfo.Summary.
Unfortunately there is a bug in HS Core (HS-1078)  which results in the Device Type String to be always blank for HS4 devices even if you set the Summary property. This bug has been fixed and the fix will be included in the next release

**Comment by spudwebb** [comment]
Fixed in HS 4.1.11.0
